### ISSUE TITLE
Quick reply sometimes don't appear.

### ISSUE THREAD

**Issue by RamyAlshurafa** [techforbetter/nightingale#147]
![j](https://user-images.githubusercontent.com/34629478/52919060-e7020380-3306-11e9-9cce-9de61fc8a4d0.png)

**Comment by thejoefriel** [comment]
Yeah this seems to be that it can't connect to the webhook fulfillment for some reason now?
